quint/io-cli-tests.md

```diff
@@ -686,6 +689,35 @@ exit $exit_code
 ]
 ```
 
+### Test does not mean assignments (#1133)
+
+See: https://github.com/informalsystems/quint/issues/1133
+
+FIXME
+
+<!-- !test in test1133 -->
+```
+output=$(quint test --match='(t1|t2)' --output='out_{#}_{}.itf.json' \
+  ./testFixture/simulator/lastActionInRun.qnt)
+exit_code=$?
+echo "BEGIN"
+# This test should have 3 states (FIXME: it does not!)
+cat out_0_t1.itf.json | jq '.states' | grep "s" | wc -l | grep 3
+rm out_0_t1.itf.json
+# This test should have 4 states (FIXME: it does not!)
+cat out_1_t2.itf.json | jq '.states' | grep "s" | wc -l | grep 4
+rm out_1_t2.itf.json
+echo "END"
+exit $exit_code
```

Comment:
Since this is not actually testing the output but just the correct construction of the trace, should this perhaps just be in the typescript tests?

In otherwords, given the parsing logic required and what it being tested, I don't understand how/why its a good fit for a black box test.

Reply:
It is testing the actual JSON file, which @ivan-gavran was inspecting by hand